codeSnippets/snippets/client-sse/src/main/kotlin/com.example/Application.kt

```diff
@@ -0,0 +1,27 @@
+package com.example
+
+import io.ktor.client.*
+import io.ktor.client.plugins.sse.*
+import kotlinx.coroutines.*
+import kotlin.time.Duration.Companion.milliseconds
+
+
+fun main() {
+
+    val client = HttpClient() {
+        install(SSE) {
+            reconnectionTime = 3000.milliseconds
```

Comment:
Actually, there is no reconnection on the client side [KTOR-6242](https://youtrack.jetbrains.com/issue/KTOR-6242). Maybe it's better to use it in the example after implementation?

Reply:
Is it that reconnection is not possible in general or only the `reconnectionTime` cannot be changed?

Reply:
Not possible in general in our implementation right now, though in plans

Reply:
Ok, I've added a comment to revisit this before the release